### ISSUE TITLE
Outdated wakatime-cli version?

### ISSUE THREAD

**Issue by LucasFA** [wakatime/vim-wakatime#162]
I found this lines in the code:

```vim
            " use Powershell to install wakatime-cli because NeoVim doesn't
            " come installed with Python bundled (https://github.com/wakatime/vim-wakatime/issues/147)
            if s:IsWindows()
                echo "Downloading wakatime-cli to ~/.wakatime/... this may take a while but only needs to be done once..."

                let cmd = 'if ((Get-WmiObject win32_operatingsystem | select osarchitecture).osarchitecture -eq "64-bit") { Write "amd64" } else { Write "386" }'
                let arch = s:Chomp(system(['powershell.exe', '-noprofile', '-command'] + [cmd]))

                let url = "https://github.com/wakatime/wakatime-cli/releases/download/v1.60.4/wakatime-cli-windows-" . arch . ".zip"
                let zipfile = s:home . "/.wakatime/wakatime-cli.zip"
```
(ignore the spacing, this was indented)

The url variable seems to be a hard coded version of wakatime-cli

**Comment by alanhamlett** [comment]
Fixed with ace2eab864609021bf9051d870073c5dd0304b27.

**Comment by LucasFA** [comment]
oh that was fast. Thanks

I do know that https://github.com/wakatime/wakatime-cli/releases/latest redirects to https://github.com/wakatime/wakatime-cli/releases/download/v1.73.1 (ie current latest release). 
I would imagine then that 
https://github.com/wakatime/wakatime-cli/releases/download/latest
redirects to the download page of the latest release

**Comment by alanhamlett** [comment]
You're right, but for some reason with the release assets `download/latest` is reversed.

Fixed with 018fa9a80c27ccf2a8967b9e27890372e5c2fb4f.
